docs/GPU-Windows.rst

```diff
@@ -270,6 +277,7 @@ Installing CMake requires one download first and then a lot of configuration for
 .. image:: ./_static/images/screenshot-downloading-cmake.png
    :align: center
    :target: ./_static/images/screenshot-downloading-cmake.png
+   :alt: A screenshot of the binary distributions of c-make for downloading on Windows 64 bit
```

Comment:
```suggestion
   :alt: A screenshot of the binary distributions of C Make for downloading on 64 bit Windows
```

I think the way you separated this out into `C` and `Make` will be better understood by screen readers.